src/schemata/definitions/step/command-handler.yml

```
@@ -0,0 +1,10 @@
+---
+type: object
+title: Command Handler
+description: |
+   A Command handler always consists of an [action](/schemas/definitions/workflow-template/actions/).
+required: [do]
+additionalProperties: false
+properties:
+  do:
+    $ref: '#/definitions/workflow-template/actions'
```

lets add `and` and `else`.. would be useful for conditional nested invocation..
```suggestion
  and:
    $ref: '#/definitions/script'
  do:
    $ref: '#/definitions/workflow-template/actions'
  else:
    $ref: '#/definitions/workflow-template/actions'
```